From f4ea011cd0387aa577236d53900de05b3c9d3fca Mon Sep 17 00:00:00 2001
From: Xu Yilun <yilun.xu@intel.com>
Date: Thu, 24 Jun 2021 13:34:30 +0800
Subject: [PATCH] fpga: dfl: fme: fix kernel-doc comments for some functions

lkp reported 2 build warnings:

   drivers/fpga/dfl/dfl-fme-pr.c:175: warning: Function parameter or member 'feature' not described in 'dfl_fme_create_mgr'

>> drivers/fpga/dfl/dfl-fme-pr.c:280: warning: expecting prototype for
>> dfl_fme_destroy_bridge(). Prototype was for dfl_fme_destroy_bridges()
>> instead

Fixes: 29de76240e86 ("fpga: dfl: fme: add partial reconfiguration sub feature support")
Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Xu Yilun <yilun.xu@intel.com>
---
 drivers/fpga/dfl-fme-pr.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/fpga/dfl-fme-pr.c b/drivers/fpga/dfl-fme-pr.c
index 619db98b7011..2d1f7718e883 100644
--- a/drivers/fpga/dfl-fme-pr.c
+++ b/drivers/fpga/dfl-fme-pr.c
@@ -158,6 +158,7 @@ static int fme_pr(struct platform_device *pdev, unsigned long arg)
  * dfl_fme_create_mgr - create fpga mgr platform device as child device
  * @feature: sub feature info
  * @pdata: fme platform_device's pdata
+ * @feature: the dfl fme PR sub feature
  *
  * Return: mgr platform device if successful, and error code otherwise.
  */
@@ -265,7 +266,7 @@ static void dfl_fme_destroy_bridge(struct dfl_fme_bridge *fme_br)
 }
 
 /**
- * dfl_fme_destroy_bridges - destroy all fpga bridge platform device
+ * dfl_fme_destroy_bridges - destroy all fpga bridge platform devices
  * @fdata: fme feature dev data
  */
 static void dfl_fme_destroy_bridges(struct dfl_feature_dev_data *fdata)
